README.md

```diff
@@ -1,135 +1,223 @@
-# Cronitor
+# Cronitor Ruby Library
 
 ![Test](https://github.com/cronitorio/cronitor-ruby/workflows/Test/badge.svg)
 [![Gem Version](https://badge.fury.io/rb/cronitor.svg)](https://badge.fury.io/rb/cronitor)
 
-[Cronitor](https://cronitor.io/) is a service for heartbeat-style monitoring of just about anything that can send an HTTP request.
 
-This gem provides a simple abstraction for the creation and pinging of a Cronitor monitor. For a better understanding of the API this gem talks to, please see [How Cronitor Works](https://cronitor.io/help/how-cronitor-works).
+[Cronitor](https://cronitor.io/) provides dead simple monitoring for cron jobs, daemons, data pipelines, queue workers, and anything else that can send or receive an HTTP request. The Cronitor Ruby library provides convenient access to the Cronitor API from applications written in Ruby.
 
-## Installation
+## Documentation
+See our [API docs](https://cronitor.io/docs/api) for a detailed reference information about the APIs this library uses for configuring monitors and sending telemetry pings.
 
-Add this line to your application's Gemfile:
+## Installation
 
-```ruby
-gem 'cronitor'
+```
+pip install cronitor
 ```
 
-And then execute:
+## Usage
 
-    $ bundle
+The package needs to be configured with your account's `API key`, which is available on the [account settings](https://cronitor.io/settings) page. You can also optionally specify an `Api Version` (default: account default) and `Environment` (default: account default).
 
-Or install it yourself as:
+These can be supplied using the environment variables `CRONITOR_API_KEY`, `CRONITOR_API_VERSION`, `CRONITOR_ENVIRONMENT` or set directly on the cronitor object.
 
-    $ gem install cronitor
+```ruby
+require 'cronitor'
+Cronitor.api_key = 'apiKey123'
+Cronitor.api_version = '2020-10-01'
+Cronitor.environment = 'staging'
+```
 
-## Usage
+You can also use a YAML config file to manage all of your monitors (_see Create and Update Monitors section below_). The path to this file can be supplied using the enviroment variable `CRONITOR_CONFIG` or call `cronitor.read_config()`.
 
+```ruby
+import cronitor
+cronitor.read_config('./path/to/cronitor.yaml')
+```
 
-### Configure
 
-You need to set Cronitor Token in order to create a monitor
+### Monitor Any Block
+```
```

Comment:
We should set the language to `ruby` here, so GitHub provides nice syntax highlighting.